index.html

```diff
@@ -232,6 +232,7 @@ <h2 class="success__title">Изображение успешно загруже
   </template>
 
   <script src="nouislider/nouislider.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/lodash@4.17.20/lodash.min.js"></script>
```

Comment:
Тут ты подключаешь lodash целиком, а используешь оттуда только одну функцию. Предлагаю вообще убрать его и подключить простой debounce вроде такого:
```suggestion
  function debounce(f, ms) {

  let isCooldown = false;

  return function() {
    if (isCooldown) return;

    f.apply(this, arguments);

    isCooldown = true;

    setTimeout(() => isCooldown = false, ms);
  };

}
```